<?xml version="1.0" encoding="UTF-8"?>
<class xmlns="http://xml.phpdox.net/src" full="App\Containers\Stripe\Exceptions\StripeApiErrorException" namespace="App\Containers\Stripe\Exceptions" name="StripeApiErrorException" abstract="false" final="false" start="13" end="19">
  <file path="/opt/lampp/htdocs/tabgo/app/Containers/Stripe/Exceptions" file="StripeApiErrorException.php" realpath="/opt/lampp/htdocs/tabgo/app/Containers/Stripe/Exceptions/StripeApiErrorException.php" size="418" time="2017-09-08T07:35:32+02:00" unixtime="1504848932" sha1="e634af3880661b0ff25190c81c977ee9043fd4c7" relative="Stripe/Exceptions/StripeApiErrorException.php"/>
  <docblock>
    <description compact="Class StripeApiErrorException."/>
    <author value="Mahmoud Zalt &lt;mahmoud@zalt.me&gt;"/>
  </docblock>
  <extends full="App\Ship\Parents\Exceptions\Exception" namespace="App\Ship\Parents\Exceptions" name="Exception" unresolved="true"/>
  <member name="httpStatusCode" default="" type="{unknown}" constant="SymfonyResponse::HTTP_EXPECTATION_FAILED" visibility="public" static="false" line="16"/>
  <member name="message" default="'Stripe API error.'" type="string" visibility="public" static="false" line="18"/>
</class>
